openshift/pipelines/scripts/common.groovy

```diff
@@ -10,24 +10,33 @@ def version = "1.0"
 
 // Run Tests
 def runFrontendTests() {
-  // pull code
-  checkout scm
   timeout(20) {
     dir('frontend') {
-      sh 'node --version'
-      sh 'npm --version'
       echo 'Installing NPM Dependencies...'
       sh 'npm ci'
-      echo 'Reporting Outdated and Vulnerable Dependencies...'
+      echo 'Reporting Vulnerable Dependencies...'
       sh 'npm audit || true'
-      sh 'npm outdated || true'
```

Comment:
We don't do anything with outdated dependencies so I removed this check here. GitHub already provides a `dependabot` that alerts you when you have a vulnerable dependency that needs updating. Not the same but good enough. 
I'm okay to restore this if others think this is of value, but I have not seen that we regularly review and update dependencies

Reply:
I think dependabot is good enough presently.